slides/src/content/two-way-data-binding.md

```diff
@@ -0,0 +1,130 @@
+<!-- .slide: id="two-way-data-binding" -->
+## Two-Way Data Binding
+
+- Combination of an `@Input` with an `@Output` using the *banana in a box* syntax `[(event)]`
+- The name of the event has to be equal to the name of the input plus the suffix "Change".
+
+```html
+<app-to-do-list [thingsToDo]="thingsToDo" (thingsToDoChange)="thingsToDo=$event"></app-to-do-list>
+```
+
+Is equivalent to:
+
+```html
+<app-to-do-list [(thingsToDo)]="thingsToDo"></app-to-do-list>
+```
+
+The built-in directive `NgModel` uses this trick to behave similar to Angular 1:
+
+```html
+<input [(ngModel)]="name" />
+```
+
+Which is equivalent to:
+
+```html
+<input [ngModel]="name" (ngModelChange)="name=$event" />
+```
+
+---
+<!-- .slide: id="two-way-data-binding-example-1" -->
+## Two-Way Data Binding Example
+
+- Let's add a remove button to demonstrate two-way databinding
+
+#####_src/app/to-do-list/to-do-list.component.html
```

Comment:
Missing underscore at the end of the line leaves everything in italics - did you preview?